modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -4451,7 +4451,7 @@ The period during which disk usage is checked for disk pressure, and data is opt
 
 *Accepted values:* [`-17592186044416`, `17592186044415`]
```

Comment:
Should we make the accepted values start from 1? 
(for some duration properties, negative duration is accepted)